config-rbac.yaml

```diff
@@ -12,6 +12,9 @@ rules:
   - apiGroups: [""]
     resources: ["pods", "pods/status", "pods/log", "pods/exec", "jobs", "jobs/status", "jobs/log"]
     verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: ["extensions", "apps"]
+    resources: ["deployments"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
@pditommaso , the verbs are specified here.

The GH comment interface makes it a bit obscure to scroll down.

Reply:
OK!

Reply:
let me add to out EKS

Reply:
I've added this 

```
  - apiGroups: ["extensions", "apps"]
    resources: ["deployments"]
    verbs: ["get", "list", "watch", "create", "delete"]
```


Have you test it on Eks or your local k8s ?

Reply:
On EKS, I am having auth issues.

On my own cluster yes, this is working and it helped resolve the testing for K8s spec updates PR https://github.com/seqeralabs/nf-tower-cloud/pull/1735#pullrequestreview-688321766

Reply:
> On EKS, I am having auth issues related to AWS permissions (as mentioned in our DM on slack).

Think now should work, because I've added the above

Reply:
Hmm, I'm still facing this issue on my end (perhaps due to AWS security/access layer and not because of k8s one 🤔 )

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/12799326/122763159-4f2dcf00-d29e-11eb-8d9e-f569389a6a71.png">

Reply:
using the aws key I've shared with you it worked for me 🤷‍♂️